Docs/introducing-blazor-forms.md

```diff
@@ -0,0 +1,215 @@
+# Introducing Blazor Forms
+
+Forms are a key component in any web application as they allow us to collect information from end-users.  This could range from collecting user details as part of a sign-up process through to a single textbox that allows users to leave comments or post messages in an application.
+
+In this article we'll explain the benefits of Blazor forms, we'll look at their lifecycle and we'll explain the fundamental concepts used to create form-driven solutions using Blazor.
+
+## Form Fundamentals
+
+Let's assume that we are building a Contact form which has two fields (Name and Email) and a button to submit the form. The code required to do this in ASP.NET Core would look similar to this.
+
+```html
+<form asp-controller="Home" asp-action="Register" method="post"> 
+    <label asp-for="Contact.Name" class="form-label">Name</label> 
+    <input asp-for="Contact.Name" class="form-control" type="text" /> 
+    <span asp-validation-for="Contact.Name" class="text-danger"></span> 
+
+    <label asp-for="Contact.Email" class="form-label">Email address</label> 
+    <input asp-for="Contact.Email" class="form-control" /> 
+    <span asp-validation-for="Contact.Email" class="text-danger"></span> 
+
+    <button class="btn btn-primary mt-3" type="submit">Submit</button> 
+</form>
+```
+
+The form defines an HTTP method and controller action to which the data is sent. For brevity, the Controller code which handles the form submission is omitted.
+
+The form uses ASP.NET tag helpers to render the correct types of input controls and validation messages based on attributes configured on the underlying data model.
+
+One problem with this solution is that when the form is submitted, a full page refresh is required to update the state of the page. This requires sending a full HTTP payload to the client and it interrupts the users' experience. For example, the newly loaded page may result in the scroll position changing thus requiring the user to scroll to find where they left off!
```

Comment:
Replace "scroll to find where they left off" with "scroll to where they left off"

Reply:
Sure thing. This is fixed now.